youtube_dl/extractor/theintercept.py

```diff
@@ -0,0 +1,50 @@
+# encoding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+
+class TheInterceptIE(InfoExtractor):
+    _VALID_URL = r'https://theintercept.com/fieldofvision/(?P<id>.+?)/'
+    _TESTS = [{
+        'url': 'https://theintercept.com/fieldofvision/thisisacoup-episode-four-surrender-or-die/',
+        'info_dict': {
+            'id': 'thisisacoup-episode-four-surrender-or-die',
+            'ext': 'mp4',
+            'title': '#ThisIsACoup – Episode Four: Surrender or Die',
+            'upload_date': '20151218',
+            'description': 'md5:74dd27f0e2fbd50817829f97eaa33140',
+        }
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        video_id = mobj.group('id')
+        webpage = self._download_webpage(url, video_id)
+        title = self._og_search_title(webpage).split(' — ')[0]
+        description = self._og_search_description(webpage)
+        thumbnail = self._og_search_property('image', webpage)
+        content_id = thumbnail.split('/')[-1].split('.')[0]
```

Comment:
better info is available in `window.initialStoreTree`.
```
"46214": {
        "ID": 46214,
        "title": "#ThisIsACoup – Episode Four: Surrender or Die",
        "slug": "thisisacoup-episode-four-surrender-or-die",
        "excerpt": "Despite a historic Twitter protest, an austerity deal is signed. In an exclusive interview, Alexis Tsipras explains how it went so wrong.",
        "content": "\u003cp class=\"p1\">\u003cspan class=\"s1\">Despite a historic twitter protest involving up to a billion people, a new austerity deal is signed. In an exclusive first English interview since the climb-down Tsipras gives his own explanation of why things went wrong.\u003c/span>\u003c/p>\n\u003cp class=\"p1\">\u003cspan class=\"s1\">Catch up on earlier episodes of #ThisIsACoup:\u003cbr />\n\u003c/span>\u003cspan class=\"s2\">\u003ca href=\"https://theintercept.com/fieldofvision/episode-one-angela-suck-our-balls/\">Episode One: &#8220;Angela, Suck Our Balls&#8221;\u003cbr />\n\u003c/a>\u003c/span>\u003cspan class=\"s2\">\u003ca href=\"https://theintercept.com/fieldofvision/episode-two-to-pay-or-not-to-pay/\">Episode Two: To Pay Or Not To Pay?\u003cbr />\n\u003c/a>\u003c/span>\u003cspan class=\"s1\">\u003ca href=\"https://theintercept.com/fieldofvision/thisisacoup-episode-three-oxi-the-greek-word-for-no/\">Episode Three: OXI &#8211; The Greek Word for “No”\u003c/a>\u003c/span>\u003c/p>\n",
        "link": "https://theintercept.com/fieldofvision/thisisacoup-episode-four-surrender-or-die/",
        "thumbnail": null,
        "date": "2015-12-18T09:00:39+00:00",
        "authors": [{
          "ID": "45802",
          "display_name": "Theopi Skarlatos",
          "url": "https://theintercept.com/staff/theopi-skarlatos/",
          "brief_bio": "",
          "full_bio": "",
          "email": "",
          "image_350": "https://theintercept.com/wp-includes/images/blank.gif",
          "twitter": "",
          "facebook": "",
          "pgp_key": "",
          "pgp_file": "",
          "pgp_fingerprint": "",
          "user_title": "",
          "user_group": "",
          "nicename": "theopi-skarlatos"
        }, {
          "ID": "45803",
          "display_name": "Paul Mason",
          "url": "https://theintercept.com/staff/paul-mason/",
          "brief_bio": "",
          "full_bio": "",
          "email": "",
          "image_350": "https://theintercept.com/wp-includes/images/blank.gif",
          "twitter": "",
          "facebook": "",
          "pgp_key": "",
          "pgp_file": "",
          "pgp_fingerprint": "",
          "user_title": "",
          "user_group": "",
          "nicename": "paul-mason"
        }],
        "excerpt_type": "",
        "is_feature": "",
        "feature_title": "",
        "is_blog": false,
        "blog_title": "",
        "comments_number": "17",
        "bitly_url": "",
        "comments_open": true,
        "is_banner_hidden": false,
        "featured_video_url": "",
        "meta": {
          "links": []
        },
        "layout": "",
        "banner_videokey": "",
        "post_type": "fov_video",
        "share_twitter_text": "",
        "yoast_wpseo_canonical": "https://theintercept.com/fieldofvision/thisisacoup-episode-four-surrender-or-die/",
        "yoast_wpseo_general_title": "",
        "yoast_wpseo_general_description": "",
        "yoast_wpseo_facebook_title": "",
        "yoast_wpseo_facebook_description": "",
        "yoast_wpseo_facebook_thumbnail": "",
        "yoast_wpseo_twitter_title": "",
        "yoast_wpseo_twitter_description": "",
        "yoast_wpseo_twitter_thumbnail": "",
        "fov_videoid": "yqkRn00b",
        "fov_teaservideoid": "IKeZfZFH",
        "fov_istrailer": "0",
        "fov_videolength": "16 min.",
        "fov_fieldnotesid": "",
        "fov_ispromoted": "1"
      }
```